src/test/scala/com/cognite/sdk/scala/v1/RawTest.scala

```diff
@@ -183,6 +183,13 @@ class RawTest extends SdkTestSpec with ReadBehaviours with WritableBehaviors wit
           .toList
           .length === 2
       )
+
+      rows.retrieveByKey("abc") match {
+        case RawRow(key, columns, _) => {
+          assert(key === "abc")
+          assert(columns === Map("a" -> "0".asJson, "abc" -> Map("cde" -> 1).asJson))
+          }
```

Comment:
Maybe add a `case _ => assert(false)` (or something like it, but with a nicer error message) so tests won't just fail with a match error if a row with that key isn't found.

Reply:
this is always match `RawRow` (`rows.retrieveByKey("abc")` returns `Id[RawRow]`)

If the row is not found, the request fails and you get

```
com.cognite.sdk.scala.common.CdpApiException: Request with id db33b322-c079-90c5-9574-2bfc35365b62 to https://api.cognitedata.com/api/v1/projects/playground/raw/dbs/raw-test-1635da95/tables/raw-test-18869257/rows/z failed with status 404: Did not find any rows with that key.
```